packages/language-server/src/completion-items.ts

```diff
@@ -281,6 +338,35 @@ function getClassNamespacePrefix(
   return undefined;
 }
 
+function getAddNamespaceEdit(
+  xmlElement: XMLElement,
+  xmlns: string,
+  value: string
+): TextEdit {
+  // Add the namespace to the top-level element
+  let parent: XMLElement | XMLDocument = xmlElement;
+  while (parent.type !== "XMLDocument") {
+    parent = parent.parent;
+  }
+  let positionLine = 0;
+  let positionColumn = 0;
+  // The 'else' part will never happen because to get suggestions on class name the open tag must exist so the
+  // xml document will not be empty
+  /* istanbul ignore else */
+  if (parent.rootElement !== null) {
```

Comment:
lets avoid doing any text edit at runtime if our data/xml/model is invalid.

Reply:
Done